sync2jira/upstream_issue.py

```diff
@@ -34,6 +34,25 @@
 
 
 log = logging.getLogger('sync2jira')
+graphqlurl = 'https://api.github.com/graphql'
+project_items_query = '''query MyQuery($orgname: String!, $reponame: String!,
+                            $ghfieldname: String!, $issuenumber: Int!) {
+    repository(owner: $orgname, name: $reponame) {
+        issue(number: $issuenumber) {
+            title
+            body
+            projectItems(first: 1) {
+                nodes {
+                    fieldValueByName(name: $ghfieldname) {
+                        ... on ProjectV2ItemFieldNumberValue {
+                            number
+                        }
+                    }
+                }
+            }
+        }
+    }
+}'''
```

Comment:
The formatting here seems slightly strange.  If GraphQL really doesn't care about whitespace, I recommend something like this:
```python
project_items_query = '''
    query MyQuery(
        $orgname: String!, $reponame: String!, $ghfieldname: String!, $issuenumber: Int!
    ) {
        repository(owner: $orgname, name: $reponame) {
            issue(number: $issuenumber) {
                title
                body
                projectItems(first: 1) {
                    nodes {
                        fieldValueByName(name: $ghfieldname) {
                            ... on ProjectV2ItemFieldNumberValue {
                                number
                            }
                        }
                    }
                }
            }
        }
    }
'''
```

Reply:
Done. 1afad545ee80433015e30fc217161ccda367d316